README.md

```diff
@@ -39,50 +39,78 @@ Update the application's configuration class to expose accessor methods for `Pac
 
     public class MySecureConfiguration extends Configuration {
         @NotNull
-        Pac4jFactory pac4jFactory = new Pac4jFactory();
+        Pac4jConfiguration pac4jConfiguration = new Pac4jConfiguration();
 
         @JsonProperty("pac4j")
-        public Pac4jFactory getPac4jFactory() {
-            return pac4jFactory;
+        public Pac4jConfiguration getPac4jConfiguration() {
+            return pac4jConfiguration;
         }
 
         @JsonProperty("pac4j")
-        public void setPac4jFactory(Pac4jFactory pac4jFactory) {
-            this.pac4jFactory = pac4jFactory;
+        public void setPac4jConfiguration(Pac4jConfiguration pac4jConfiguration) {
+            this.pac4jConfiguration = pac4jConfiguration;
         }
     }
 
+Note that it is also possible to have `pac4jConfiguration` be null and in this case, the bundle will be disabled.
```

Comment:
Ah yes, I set up editorconfig with a wrapping at 120 characters, I will change that.